nixos/modules/services/desktops/pipewire.nix

```diff
@@ -60,28 +61,26 @@ in {
   config = mkIf cfg.enable {
     assertions = [
       {
-        assertion = cfg.pulse.enable -> !config.hardware.pulseaudio.enable;
-        message = "PipeWire based PulseAudio emulation doesn't use the PulseAudio service";
+        # TODO: it is also possible to listen on a separate socket in which case
+        # pipewire and pulse could be run side by side
```

Comment:
I don't think we need to support that use-case (at least for now). Seems very advanced/non-standard/etc. to me.